DASCharter-2020.html

```diff
@@ -960,8 +956,6 @@ <h3>
               "https://github.com/web-platform-tests/wpt/issues?utf8=%E2%9C%93&amp;q=label%3Atype%3Auntestable%20">type:untestable</a>'
               label to follow up later.
             </p>
-          </blockquote>
```

Comment:
I think we need to keep the closing tag.

Reply:
Fixed. Thank you for catching this!